source/includes/_tipser-elements.md

```diff
@@ -102,6 +102,77 @@ All properties are optional:
 property name | description |type | default value
 --------------|-------------|-----|---------------
 defaultAddedToCartPopup | controls whether default Added To Cart Popup is displayed | boolean | true 
+eventsHandlers | gives possibility to add handlers to Tipser Elements events | object | empty
```

Comment:
instead of empty I would suggest to type `{}`

Reply:
As I did for instance here: 
<img width="770" alt="Zrzut ekranu 2019-05-9 o 12 00 40" src="https://user-images.githubusercontent.com/3996881/57445187-15caae80-7252-11e9-903e-2f9ebdeda19e.png">

Reply:
Yes, I think `{}` is very understandable for readers with JS background and is more explicit than "empty".

But @chmurson please make sure to update your docs with type = object instead of type = {} ;-)

Reply:
> But @chmurson please make sure to update your docs with type = object instead of type = {} ;-)

+1